vta/python/vta/pkg_config.py

```diff
@@ -109,7 +109,20 @@ def __init__(self, cfg, proj_root):
         #   - axi_cache_bits:   ARCACHE/AWCACHE signals for the AXI bus
         #                       (e.g. 1111 is write-back read and write allocate)
         #   - axi_prot_bits:    ARPROT/AWPROT signals for the AXI bus
-        if self.TARGET == "ultra96":
+        if self.TARGET == "de10nano":
+            self.fpga_device = "5CSEBA6U23I7"
+            self.fpga_family = "Cyclone\\ V"
```

Comment:
At the moment, the parameters below are not used in the hardware generation, correct?

One option is to propagate these as we do in the HLS generation scripts (see how these parameters get passed in). An alternative is to simply add a comment on the side of the lines below stating that these parameters are currently not propagated as a TODO.

Reply:
I admit the remaining parameters have not been used currently. I think enabling the parameterization of all these would be a future change, so I have chosen the option to make a comment as a TODO.